canonicalwebteam/search/views.py

```diff
@@ -46,18 +44,16 @@ def build_search_view(
         )
     """
 
+    app = flask.Flask(__name__)
+    limiter = Limiter(
+        get_remote_address, app=app, default_limits=[request_limit]
+    )
+
+    @limiter.limit(request_limit)
```

Comment:
This implementation doesn't work, the moment I removed this decorator, everything passed locally. So I came up with another better implementation:
```python
with limiter.limit(request_limit):
  flask.render_template(
                    template_path,
                    query=query,
                    start=start,
                    num=num,
                    results=results,
                    siteSearch=site_search,
                ),
                {"X-Robots-Tag": "noindex"},
            )
 ```
 
 This seems to pass the tests